src/K1DModel/tendency.jl

```diff
@@ -80,6 +80,89 @@ end
     @. aux.activation_sources = to_sources(-S_Nl, S_Nl)
 end
 
+# helper functions for precomputing aux activation for cloudy
+@inline function get_aerosol_tendency(
+    kid_params,
+    thermo_params,
+    air_params,
+    activation_params,
+    q_tot,
+    q_liq,
+    N_liq,
+    N_aer,
+    T,
+    p,
+    ρ,
+    ρw,
+    dt,
+)
+    FT = eltype(q_tot)
+    S_Nl::FT = FT(0)
+    S_Na::FT = FT(0)
+
+    q = TD.PhasePartition(q_tot, q_liq, FT(0))
+    S::FT = TD.supersaturation(thermo_params, q, ρ, T, TD.Liquid())
+
+    (; r_dry, std_dry, κ) = kid_params
+    w = ρw / ρ
+
+    aerosol_distribution =
+        CMAM.AerosolDistribution((CMAM.Mode_κ(r_dry, std_dry, N_aer, (FT(1),), (FT(1),), (FT(0),), (κ,)),))
+    N_act = CMAA.N_activated_per_mode(activation_params, aerosol_distribution, air_params, thermo_params, T, p, w, q)[1]
```

Comment:
This is out of date with the aerosol activation PR that I merged, apologies!

Reply:
Can we use this one: `CMAA.total_N_activated` ? Or does it have to be only one aerosol mode for Cloudy right now?